### ISSUE TITLE
[function name not correct after load pdb][rename function by label]

### ISSUE THREAD

**Issue by djerryz** [NationalSecurityAgency/ghidra#4883]
Ghidra(10.2.2) load sqlchiper.dll and pdb, after analysis ,  the function name will become like `_func__cdecl_void`, This is very confusing!
for example, the source code is:
```C
int sqlite3_prepare_v2(
   parameters_here
){
  rc = sqlite3LockAndPrepare(db,zSql,nBytes,SQLITE_PREPARE_SAVESQL,0,
                             ppStmt,pzTail);
}
```
by check `sqlite3_prepare_v2` in ghidra , i will look like:
![image](https://user-images.githubusercontent.com/24947060/211492810-d7776591-409f-4b81-8a7e-5f77c0430157.png)

i think `_func__cdecl_int_sqlite3_ptr_char_ptr_int_uint_Vdbe_ptr_sqlite3_stmt_ptr_ptr_char_ptr_ptr` is function `sqlite3LockAndPrepare`, so i find `sqlite3LockAndPrepare`, i get this:
![image](https://user-images.githubusercontent.com/24947060/211493094-d8ec9e86-4d1c-4e63-9fe8-68f606cd844e.png)

you can see, `sqlite3LockAndPrepare` just can be find in `Labels` but not in `Functions`.
i think label  `sqlite3LockAndPrepare` is map to `_func__cdecl_int_sqlite3_ptr_char_ptr_int_uint_Vdbe_ptr_sqlite3_stmt_ptr_ptr_char_ptr_ptr`, so i  try to rename `_func__cdecl_int_sqlite3_ptr_char_ptr_int_uint_Vdbe_ptr_sqlite3_stmt_ptr_ptr_char_ptr_ptr` as `sqlite3LockAndPrepare`, but there are almost Thousands of similar situations, Manual operation one by one is not realistic, so is there a way to Fast and automatic achieve.

Also, i compare with IDA Pro 7.4, load dll and pdb, here is it:
![image](https://user-images.githubusercontent.com/24947060/211494131-71591ac7-1a11-4659-ba3e-cc315460e0b8.png)

also in bindiff:
![image](https://user-images.githubusercontent.com/24947060/211494413-d0faa058-31f1-4204-9bff-808cf5c324c2.png)

so, is function name not correct after load pdb ? and is there a quick a way to rename function by label? thanks.

i upload sqlchiper.dll and sqlchiper.pdb in this zip file: [sqlcipher.zip](https://github.com/NationalSecurityAgency/ghidra/files/10380364/sqlcipher.zip)

**Comment by ryanmkurtz** [comment]
Duplicate of #4842 and #4873?

**Comment by ryanmkurtz** [comment]
This should be fixed already in 10.2.3.
